digital_printing/digital_printing/doctype/print_order/print_order.py

```diff
@@ -158,6 +164,84 @@ def calculate_totals(self):
 	def on_upload_complete(self):
 		self.set_missing_values()
 
+	@frappe.whitelist()
+	def create_printed_design_item(self):
+		if self.docstatus != 1:
+			frappe.throw(_("Submit the document first."))
+
+		for d in self.items:
+			if d.item_code:
+				continue
+
+			item_doc = frappe.get_doc({
+				"doctype": "Item",
+				"item_naming_by": "System Generated",
```

Comment:
Lets remove this line. And let the Meta's default decide whether it should be Item Code or Item Name. Lets also decide on a fallback Item Code